feature-libs/product-configurator/common/components/configure-product/configure-product.component.html

```diff
@@ -11,7 +11,8 @@
         } | cxUrl
       "
       [queryParams]="{
-        displayRestartDialog: isDisplayRestartDialog(product.configuratorType)
+        displayRestartDialog: isDisplayRestartDialog(product.configuratorType),
+        productCode: product.code
```

Comment:
`productCode` could be added to url by clicking on **Configure** button but it is not required